File: arrays 101/max_consecutive_ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int i;
        int large=0;
        int count=0;
        for(i=0; i<nums.length; i++)
        {
            
            if(nums[i]==1)
            {
                count++;
            }
            
            else if(nums[i]==0)
            {
                count=0;
            }
            if(count>large)
            {
                large=count;
            }
        }
        
        return large;
    }
}

File: arrays 101/max_consecutive_ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        large=count
        for i in range(len(nums)):
            if bin(nums[i])==bin(1):
                count+=1
            elif bin(nums[i])==bin(0):
                count=0
            if count>large:
                large=count
        return large

